src/domain/eventSearch/searchPanel/SearchPanel.tsx

```diff
@@ -72,6 +81,12 @@ const SearchPanel: React.FC = () => {
     });
   };
 
+  const handleChangePublishers = (newPublishers: OptionType[]) => {
```

Comment:
This function is duplicated three times. Maybe there could be one common utility function that takes as an additional argument the `setSearchState` function?

Reply:
I implemented an useEventSearchHelpers hook which takes setSearchState as a parameter and return different type helper functions to set new search values.